# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1498564672 14400
#      Tue Jun 27 07:57:52 2017 -0400
# Node ID b132881f9affbc1f61c0a905861c34209579bfda
# Parent  6683451b75b21742df1c16ac1180ccfd372c4f91
* ismember.m: Tag two known bug tests as fixed.

diff --git a/scripts/set/ismember.m b/scripts/set/ismember.m
--- a/scripts/set/ismember.m
+++ b/scripts/set/ismember.m
@@ -226,14 +226,14 @@ endfunction
 %! assert (result, [true; false; true]);
 %! assert (s_idx, [1; 0; 2]);
 
 %!test
 %! [result, s_idx] = ismember ([1:3; 5:7; 4:6; 0:2; 1:3; 2:4], [1:3], "rows");
 %! assert (result, logical ([1 0 0 0 1 0]'));
 %! assert (s_idx, [1 0 0 0 1 0]');
 
-%!test <51187>
+%!test <*51187>
 %! assert (ismember ('b ', {'a ', 'b '}), true);
 
-%!test <51187>
+%!test <*51187>
 %! abc = ['a '; 'b '; 'c '];
 %! assert (ismember (abc, {abc}), [false; false; false]);
